.circleci/config.yml

```diff
@@ -31,13 +20,21 @@ references:
             - node_modules
             - ~/.cache/yarn
 
+      # TODO: Remove workaround if https://github.com/GoogleChrome/puppeteer/issues/3774 was resolved
+      - run:
+          name: Workaround for broken Chrome 73 (Use Chromium supported by Puppeteer instead)
+          command: |
+            mkdir -p ~/.puppeteer && cd ~/.puppeteer
+            npm i puppeteer
+            echo "export CHROME_PATH=$(node -e "console.log(require('puppeteer').executablePath())")" >> $BASH_ENV
```

Comment:
Add workaround for broken Chrome 73. We have to use Chromium supported by Puppeteer (Chromium 74) within image.

We have confirmed to work conversion in Chrome 73 with out of container (Windows/macOS/Linux).